README.md

```diff
@@ -7,9 +7,9 @@ basic-programs/
 ├──<Language Name>/
 │  ├──<Descriptive name>.example
 ```
-If there is no folder for the language in the repository, you are more than welcome to add another.
+If there is not a folder for the language in the repository, you are more than welcome to add another.
```

Comment:
You have the previous version of this file. Just pull in the latest branch or remove this file and commit again.

Reply:
ok